Sources/Hyperspace/Service/Backend/BackendService.swift

```diff
@@ -35,19 +35,18 @@ extension BackendService: BackendServiceProtocol {
         networkService.execute(request: request.urlRequest) { result in
             switch result {
             case .success(let serviceSuccess):
-                BackendServiceHelper.handleResponseData(serviceSuccess.data, for: request, completion: completion)
+                BackendServiceHelper.handleResponseData(serviceSuccess.data, serviceSuccess: serviceSuccess, for: request, completion: completion)
             case .failure(let serviceFailure):
                 BackendServiceHelper.handleNetworkServiceFailure(serviceFailure, completion: completion)
             }
         }
     }
     
     public func execute<T: Request & Recoverable>(recoverable request: T, completion: @escaping BackendServiceCompletion<T.ResponseType, T.ErrorType>) {
-        execute(request: request) { [weak self] result in
+        execute(request: request) { [weak self] (result) in
```

Comment:
Super minor, but we probably want to remove the parens to keep the style uniform.

Reply:
I forgot to change those back